docs/how-to-guides/connecting-from-go.md

```diff
@@ -223,11 +223,21 @@ message type. We will use the wrapped ugnot realm for this example, wrapping
 `1000000ugnot` (1 $GNOT) for demonstration purposes.
 
 ```go
-msg := gnoclient.MsgCall{
-    PkgPath:  "gno.land/r/demo/wugnot", // wrapped ugnot realm path
-    FuncName: "Deposit",                // function to call
-    Args:     nil,                      // arguments in string format
-    Send:     "1000000ugnot",           // coins to send along with transaction
+import (
+    ...
+	"github.com/gnolang/gno/gno.land/pkg/gnoland/ugnot"
+	"github.com/gnolang/gno/gno.land/pkg/sdk/vm"
+	"github.com/gnolang/gno/tm2/pkg/std"
+)
+```
+
+```go
+msg := vm.MsgCall{
+    Caller:  accountRes.GetAddress(),                                 // address of the caller (signer)
```

Comment:
You can also use `addr` I think

Reply:
Good catch. I pushed a commit to change it.